lib/compiler-singlepass/src/machine_x64.rs

```diff
@@ -2180,7 +2252,11 @@ impl Machine for MachineX86_64 {
 
     fn emit_function_prolog(&mut self) {
         self.emit_push(Size::S64, Location::GPR(GPR::RBP));
+        self.unwind_ops
+            .push((self.get_offset().0, UnwindOps::PushFP { up_to_sp: 16 }));
```

Comment:
It might be better to wrap this in a `emit_unwind_op` function to avoid having to add `self.get_offset().0` every time.

Reply:
Ah yes, I'll do that.